src/Microsoft.Extensions.Logging.EventLog/Internal/IEventLog.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Diagnostics;
+
+namespace Microsoft.Extensions.Logging.EventLog.Internal
+{
+    public interface IEventLog
+    {
+        int MaxMessageSize { get; }
```

Comment:
Sure, I agree...will change it...BTW, I noticed that my tests were failing in Travis due to new line representations being different in Linux(`\n` instead of `\r\n`)...I temporarily enabled this test to run only on Windows as EventLog is a windows concept...this is because of the same reason that computing the strings(taking into account the different newline lenghts) would make the test not easy to read and itself can have bugs...